### ISSUE TITLE
plot3d("*",...) gives internal error

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#4053]
Imported from SourceForge on 2024-07-09 10:47:10
Created by **[macrakis](https://sourceforge.net/u/macrakis/)** on 2021-04-22 14:16:15
Original: https://sourceforge.net/p/maxima/bugs/3776

---

`plot2d("!",[x,0,1])` works fine.

But `plot3d("*",[x,0,1],[y,0,1]);`

gives an error:

`declare: argument must be a symbol; found *
 -- an error. To debug this try: debugmode(true);`
 
 The problem is with the operator, not with using a two-argument function -- this works fine:
 `mul(a,b):=a*b$
 plot3d(mul,[x,0,1],[y,0,1])`

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-09 10:47:11
Created by **[macrakis](https://sourceforge.net/u/macrakis/)** on 2021-04-22 16:45:27
Original: https://sourceforge.net/p/maxima/bugs/3776/#7e98

---

- Description has changed:

Diff:

~~~~

--- old
+++ new
@@ -7,7 +7,7 @@
 `declare: argument must be a symbol; found *
  -- an error. To debug this try: debugmode(true);`
  
- The problem is with the operator, not with using a two-valued function -- this works fine:
+ The problem is with the operator, not with using a two-argument function -- this works fine:
  `mul(a,b):=a*b$
  plot3d(mul,[x,0,1],[y,0,1])`
  

~~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-09 10:47:15
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2022-02-28 00:43:53
Original: https://sourceforge.net/p/maxima/bugs/3776/#d366

---

Fixed by commit f6cf9e5 to KINDP. I  put a test example into tests/rtest_plot.mac, commit 1cb3994. Closing this report as fixed.
